requirements.txt

```diff
@@ -1,9 +1,5 @@
-# Developers: Please do not modify this auto-generated file. If
-# you want to add, remove, upgrade, or downgrade libraries,
-# please change the `requirements.in` file, and then follow
-# the instructions there to regenerate this file.
```

Comment:
Yes ! It doesn't matter. I just thought having requirements.txt with 3.8 specification will be relevent since we all code will reflected python 3.8 enviornment . Having a requirements.txt generated with 3.7 version may cause some confusion . Even thought it will be auto updated when the user installs oppia .